loama/src/views/LoginView.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="panel-container">
+    <div class="left-panel">
+      <LoginForm @toggle-provider="toggleSidePanel" v-model:showPopup="showPopup" title="LOAMA"
+        subtitle="SOLID permissions made easy" />
```

Comment:
```suggestion
      <LoginForm
        @toggle-provider="toggleSidePanel"
        v-model:showPopup="showPopup"
        title="LOAMA"
        subtitle="SOLID permissions made easy"
      />
```

Reply:
Again, my formatter reformats this to a single line.

Reply:
might look into prettier settings tomorrow during the day so don't worry about it